### ISSUE TITLE
RTL Message Layout Bug

### ISSUE THREAD

**Issue by Moork0** [SchildiChat/SchildiChat-android#74]
**Describe the bug**

**To Reproduce**
1. Go to a room with RTL language(in this case Farsi) speaking members.
2. look for some messages that are written in Persian(Farsi)
3. You'll see some messages have few first characters missing.
4. ...

**Expected behavior**
As you can see in screenshots, If you reply to a corrupted message it would be fine and vice versa: you may correctly see a message but if you reply to that message, the part of your reply which shows the original message would break.

**Screenshots**
**First Case: original message looks corrupted but in the reply it looks fine.**

![firstcase1](https://user-images.githubusercontent.com/24620298/125202696-e8db1180-e289-11eb-90f4-ff08991cef3f.png)
![firstcase2](https://user-images.githubusercontent.com/24620298/125202698-ee385c00-e289-11eb-8551-46d2e5abc27b.png)

**Second Case: original message looks find but in reply message, the part showing original message is being corrupted**

![secondcase1](https://user-images.githubusercontent.com/24620298/125202777-6010a580-e28a-11eb-8f21-a322ec236735.png)
![secondcase2](https://user-images.githubusercontent.com/24620298/125202739-348dbb00-e28a-11eb-9bbe-e2d7bd1f24c5.png)

**Smartphone (please complete the following information):**
 - Device: Nokia5
 - OS: Android 10

(Other people with other devices have this issue too)

**Additional context**
 - App version and store: 1.1.11.sc36 FDROID
 - Homeserver: tchncs.de

**Upstream relevance**
 - Does this issue exist in Element Android?  No, it's particular to schildi's layout.
 - If this issue exists in Element Android, please also link the corresponding bug report (create a new one if it doesn't exist yet)

**Comment by SpiritCroc** [comment]
Would you mind pasting some text that I can copy for testing?

**Comment by Moork0** [comment]
Sure. I will paste the texts which are in the screenshots.
<div dir="rtl">

- "راستش من تو ۴ ماه دیدم جای شما تجربه اش کردم😄"
- "ببخشید منظورتون از اجتماعش سمیه چیه ؟!"
- "توصیهٔ من اینه که sftp استفاده شه."
- "سامبا افتضاح‌ترین قرارداد انتقال پرونده است. به دشمنتون هم توصیه نکنید!"

</div>

**Comment by SpiritCroc** [comment]
Some more questions:
- Which language setting are you using for the SchildiChat app and/or for your Android system? (Do I see correctly it's a LTR language?)
- Does this only happen during reply rendering, or also for messages that do not contain any quoted text?

From my first testing with your messages, my layout is broken in a different way than in your screenshots :thinking:

**Comment by Moork0** [comment]
1. Yes I use English as both SchildiChat and Android System.
2. Mostly it is happening during reply rendering. but I just checked and I can confirm it is also happen for messages that do not contain any qouted text.

I'll add some examples for this second case(messages that are not replies).

**Comment by Moork0** [comment]
Well, I found that messages. But it seems there is another bug with rendering messages with quoted text.
I saw this in SchildiChat and I thought it is an independent message. but it's not.

![schildi2](https://user-images.githubusercontent.com/24620298/125499619-0cf77eeb-c785-4526-ab33-e550058c4728.png)

**The same message in Element-web:**

![element](https://user-images.githubusercontent.com/24620298/125500028-44064029-9112-4a19-a1de-f5641186c7e5.png)


So, I think the layout breaking is only apply to messages that contain quoted text.
